src/ui/remotesignalsources.h

```diff
@@ -80,12 +82,13 @@ class QueryFilterElementList : public std::list<QueryFilterElement> {
 };
 
 class SignalList {
+    Digitizer::Settings settings;
```

Comment:
So the Settings object gets instantiated wherever we need to access some setting, each time creating a new object, reloading things from the environment. At the same time the settings object is mutable, so it looks like settings can be changed... should at least be const, maybe also with const members?

In general wouldn't it be cleaner to have one explicit app-wide settings object, that gets passed through the stack? Since this is more a question of in which direction we want to go with the API, I'd welcome further input here.

I understand that it's probably the easiest approach to get this throughj and might be acceptable for now to get things working, but i'd like to see how we want this to evolve.